src/sql/workbench/contrib/notebook/find/notebookFindModel.ts

```diff
@@ -565,9 +565,8 @@ export class NotebookFindModel extends Disposable implements INotebookFindModel
 		let searchText: string = input.substr(index);
 		while (findResults.length < maxMatches && searchText.indexOf(exp) > -1) {
 			if (wholeWord) {
-				// word in itself with no spcl characters around \\bword\\b, word ending w/ spcl character \\bword\\s
-				// word begins w/ spcl character \\sword\\b and word that begins and ends with spcl character \\sword\\s
-				let wholeWordRegex = new RegExp(`\\b${this.escapeRegExp(exp)}\\b|\\b${this.escapeRegExp(exp)}\\s|\\s${this.escapeRegExp(exp)}\\b|\\s${this.escapeRegExp(exp)}\\s`);
+				// word in itself with no spcl characters around \\bword\\b, word that begins or ends with spcl character \\sword\\s
+				let wholeWordRegex = new RegExp(`(\\b|\\s)${this.escapeRegExp(exp)}(\\b|\\s)`);
```

Comment:
nit: but you don't have to use a capture group (parens) for these. Just doing `[\\b\\s]` should accomplish the same thing

Reply:
because of the escapes in [\\b\\s] it's not working as supposed that's why I had it with (parens).

Reply:
What wasn't working? You tried [\\b\\s] instead and that didn't work? What happened? AFAIK that should be fine - this is being passed as-is to the RegExp so there shouldn't be any additional escaping being done

Reply:
yep I tried with  [\\\\b\\\\s] and that didn't work. It failed for one of my tests.